build.gradle

```diff
@@ -273,18 +273,18 @@ subprojects {
 
 
         // SLF4J as a facade over Log4j2 required dependencies
-        implementation 'org.apache.logging.log4j:log4j-api:2.16.0'
-        implementation 'org.apache.logging.log4j:log4j-core:2.16.0'
-        implementation 'org.apache.logging.log4j:log4j-slf4j-impl:2.16.0'
-        implementation 'org.apache.logging.log4j:log4j-web:2.16.0'
+        implementation 'org.apache.logging.log4j:log4j-api:2.17.0'
+        implementation 'org.apache.logging.log4j:log4j-core:2.17.0'
+        implementation 'org.apache.logging.log4j:log4j-slf4j-impl:2.17.0'
+        implementation 'org.apache.logging.log4j:log4j-web:2.17.0'
```

Comment:
It'd be better if we use the `log4j2.version` property and replace `2.17.0` references.